docs/CVMFS_kernel.md

```diff
@@ -0,0 +1,9 @@
+# Adding CVMFS Kernels
```

Comment:
In the future (but doesn't have to be this PR), it would be good to provide more of an explanation of you implemented it. We should also note that you need to setup singcvmfs on all of the HPCs, but that's probably on my plate.